<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourierViewClass</class>
 <widget class="QMainWindow" name="CourierViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CourierView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="packagesList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="zoneLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="streetsComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deliverButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deliver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>deliverButton</sender>
   <signal>clicked()</signal>
   <receiver>CourierViewClass</receiver>
   <slot>deliver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deliver()</slot>
 </slots>
</ui>
